.gitlab-ci.yml

```diff
@@ -341,6 +341,38 @@ build/cuda102/nompi/intel/cuda/debug/static:
     CUDA_ARCH: 35
 
 # cuda 11.0 and friends on HoreKa with tests
+build/cuda110/mvapich2/gcc/cuda/debug/shared:
+  extends:
+    - .build_template
+    - .default_variables
+    - .full_test_condition
+    - .use_gko-cuda110-mvapich2-gnu9-llvm9-intel2020
+  variables:
+    BUILD_OMP: "ON"
+    BUILD_CUDA: "ON"
+    BUILD_MPI: "ON"
+    BUILD_TYPE: "Debug"
+    FAST_TESTS: "ON"
+    CUDA_ARCH: 80
+    USE_NAME: "cuda110-mvapich2-gcc-${CI_PIPELINE_ID}"
+    KEEP_CONTAINER: "ON"
+    USE_SLURM: 0
+
+test/cuda110/mvapich2/gcc/cuda/debug/shared:
+  extends:
+    - .horeka_test_template
+    - .default_variables
+    - .full_test_condition
+    - .use_gko-cuda110-mvapich2-gnu9-llvm9-intel2020
+  variables:
+    USE_NAME: "cuda110-mvapich2-gcc-${CI_PIPELINE_ID}"
+    SLURM_PARTITION: "accelerated"
+    SLURM_GRES: "gpu:4"
+    SLURM_TIME: "00:45:00"
```

Comment:
I think in another pr, it exceeds the time limit.
Could you extend it to 1:00:00? also for other slurm jobs